src/hot-score.js

```diff
@@ -1,17 +1,19 @@
 
```

Comment:
Yes I think `src/core` is more appropriate location for this. We should only have directly related to the server itself under `src`